bigbluebutton-html5/imports/api/users/server/modifiers/setChangedSettings.js

```diff
@@ -0,0 +1,30 @@
+import { check } from 'meteor/check';
+import Users from '/imports/api/users';
+import Logger from '/imports/startup/server/logger';
+
+export default function setChangedSettings(userId, setting, value) {
```

Comment:
What settings are we referring to here - perhaps we can improve the function name

Reply:
I'm referring to any settings changed, what name do you suggest?